test-data/unit/fine-grained-cache-incremental.test

```diff
@@ -51,6 +51,34 @@ x = 'hi'
 ==
 a.py:3: error: Unsupported operand types for + ("int" and "str")
 
+[case testIncrCacheDoubleChange1]
```

Comment:
Would it be useful to have a test case that covers the extra dependencies, and in particular reading dependencies from parent packages? Or are there existing test cases that deal with it?

Reply:
There are existing test cases that cover it.